packages/core/src/Modules/Endpoint/Endpoint.jsx

```diff
@@ -1,24 +1,38 @@
 import React from 'react';
 import { Field, Form, Formik } from 'formik';
 import { Button, Input, Checkbox, Text } from '@deriv/components';
-import { getDebugServiceWorker, getAppId, getSocketURL, PlatformContext, isMobile } from '@deriv/shared';
+import {
+    getDebugServiceWorker,
+    getAppId,
+    getSocketURL,
+    PlatformContext,
+    isMobile,
+    website_domain,
+    TRADE_FEATURE_FLAGS,
+} from '@deriv/shared';
 import { observer, useStore } from '@deriv/stores';
 
 const FeatureFlagsSection = observer(() => {
     const { feature_flags } = useStore();
-
+    const visible_feature_flags = Object.entries(feature_flags.data ?? {})?.reduce(
+        (flags, [key, value]) => ({
+            ...flags,
+            ...(location.hostname === website_domain && TRADE_FEATURE_FLAGS.includes(key) ? {} : { [key]: value }),
+        }),
+        {} // hiding flags for trade types in production
+    );
```

Comment:
I found this a little hard to comprehend, but it could totally just be me. Would this be better?

```suggestion
const isProduction = location.hostname === website_domain;

const visibleFeatureFlags = Object
    .entries(feature_flags.data || {})
    .reduce((flags, [key, value]) => {
        if (!isProduction || !TRADE_FEATURE_FLAGS.includes(key)) {
            flags[key] = value;
        }
        return flags;
    }, {});

```